File: main/src/main/java/net/fwitz/math/main/binary/escape/newton/NewtonsMethodWikipediaExample.java
package net.fwitz.math.main.binary.escape.newton;

import net.fwitz.math.main.binary.complex.functions.examples.simple.WikipediaExample;

public class NewtonsMethodWikipediaExample {
    public static void main(String[] args) {
        NewtonsMethodPlot.render(WikipediaExample.FN_NAME + " (Newton's Method)", WikipediaExample::fn);
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/escape/root/newton/NewtonsMethodWikipediaExample.kt
package net.fwitz.math.main.binary.escape.root.newton

import net.fwitz.math.main.binary.complex.functions.examples.simple.WikipediaExample

object NewtonsMethodWikipediaExample {
    @JvmStatic
    fun main(args: Array<String>) = NewtonsMethodPlot.render(
        name = WikipediaExample.FN_NAME + " (Newton's Method)",
        f = WikipediaExample::fn
    )
}